content/admin/policy_mgmt/Source-names.md

```diff
@@ -0,0 +1,48 @@
+<!--
+title: "Source Names"
+description: "How to create and use the Source Names feature"
+tags: "admin policy management ip attacks attacker source name"
+-->
+
+Use source names to quickly identify non-threatening internal traffic and testing while monitoring attack events in your organization.
+
+## About Source Names
+
+Source names allow you to label one or more IP addresses and/or subnet masks with a display name of your choice. To create a source name, all you need is the IP information you want to mark as safe and a unique name you'll recognize in the Contrast UI. Once the source name is saved, Contrast displays it in the **Attacks > Monitor** page as well as the **Attacks Details** page instead of the user's IP information. You and other users in your organization can then quickly identify the named attacker as a known, friendly source when assessing attack events.    
+
+## Create Source Names
+
+To set up source names, go to the **user menu > Policy Management > IP Management page**, and select the **Source Names** tab. Click the button to **Add Source Name**. In the **Add Source Name** form, complete the following fields: 
```

Comment:
Should it say “to set up _a_ source name,” as you can only set up one name at a time? (As opposed to "To set up source names)

Reply:
I'd like to keep this plural to reinforce that they can have multiple names created in their org at the same time